enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -712,3 +712,103 @@ class SubsidyAccessPolicyAllocationResponseSerializer(serializers.Serializer):
             'learner email(s), and content for this action.'
         ),
     )
+
+
+class GroupMembersWithAggregatesMemberDetailsSerializer(serializers.Serializer):
```

Comment:
nit: This could just be named `GroupMembersDetailsSerializer` - there's nothing specific to aggregates in here (even though it is *consumed* by an aggregates serializer - but there's other parent serializers that might want this as a child, too).